### PR TITLE
Moved exceptions to utils file, corrected spelling errors in comments.

### PR DESCRIPTION
**Pull request by ghost**
Moved exception classes to utils file to make code reusable, corrected spelling errors in comments.

### CODE REVIEW

File: DinosaurStoryAPICalls.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 def get_random_dino_info():
     try:
         response = requests.get(get_dino_url)
-        response.raise_for_status() # If there the HTTP request returns unsecessful status code this will raise a HTTPError
+        response.raise_for_status() # If the HTTP request returns unsucessful status code this will raise a HTTPError
```
Comment (pamseale) [inline DinosaurStoryAPICalls.py]: unsuccessful

File: PokemonStoryApiCalls.py [hunk 2]
```diff
@@ -16,7 +16,7 @@ def get_pokemon_info():
 
 
         response = requests.get(url)
-        response.raise_for_status() # If there the HTTP request returns unsecessful status code this will raise a HTTPError
+        response.raise_for_status() # If the HTTP request returns unsucessful status code this will raise a HTTPError
```
Comment (pamseale) [inline PokemonStoryApiCalls.py]: unsuccessful

File: SpaceStoryAPICalls.py [hunk 3]
```diff
@@ -10,7 +10,7 @@
 def get_response():
     try:
         response = requests.get(open_notify_url)
-        response.raise_for_status() # If there the HTTP request returns unsecessful status code this will raise a HTTPError
+        response.raise_for_status() # If the HTTP request returns unsucessful status code this will raise a HTTPError
```
Comment (pamseale) [inline SpaceStoryAPICalls.py]: unsuccessful

**Review by pamseale — CHANGES_REQUESTED** [review]
successful rather than successful I'm afraid

**Review by ghost — COMMENTED** [review]
Thank you for your review Pam, I have corrected the spelling errors.

**Review by JessamineCodes — APPROVED** [review]
Looks like mostly typo fixing so looks good

**Pull request merged**